src/main/java/de/tum/cit/ase/ares/api/Policy.java

```diff
@@ -0,0 +1,39 @@
+package de.tum.cit.ase.ares.api;
+
+import org.apiguardian.api.API;
+
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+
+/**
+ * This annotation is used to specify the security policy
+ * that should be used for {@link de.tum.cit.ase.ares.api.jupiter.PublicTest}
+ * or {@link de.tum.cit.ase.ares.api.jupiter.HiddenTest}.
+ * This annotation is optional and specified in TODO.
```

Comment:
We have a TODO here, was this intended?

Reply:
This should be replaced by a link to the policy file specification as soon as we have such a link.